syntax = "proto3";

message DESIRED_POS {
    //Desired Sub coordinate orientation
    float roll   = 1;
    float pitch  = 2;
    float yaw    = 3;

    //Desired depth
    float depth  = 4;

    //Desired x and y position with respect to original (0,0) position
    float x_pos  = 5;
    float y_pos  = 6;
}
